### PR TITLE
Improve the Structure of the OpenAPI Guide

### LINKED ISSUE #4134: Improve the `Open API Support` Page
**Description:**

Need to do the following.

- [x] Reduce the length of the titles in `Open API Support` Learn page. 
- [x] Remove the content within brackets in the titles.
- [x] Remove the headings added as bullet points.

**Describe your problem(s)**

**Describe your solution(s)**

**Related Issues (optional):**

**Suggested Labels (optional):**

**Suggested Assignees (optional):**

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Improve the structure of the OpenAPI guide.
> Fixes #4134 

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

**Review by chiranSachintha — APPROVED** [review]

**Review by NipunaMadhushan — APPROVED** [review]

**Review by udda1996 — APPROVED** [review]

File: learn/guides/ballerina-openapi-support.md [hunk 1]
```diff
@@ -24,28 +24,30 @@ redirect_from:
...
-    
- > **Prerequisite**: Download and install the Ballerina Swan Lake from [downloads](https://ballerina.io/downloads/).
 
- > **Note**: Ballerina SwanLake supports OpenAPI Specification version 3.0.0 onwards.
+ > **Note**: Ballerina Swan Lake supports the OpenAPI Specification version 3.0.0 onwards.
 
-## Generate a Ballerina service from an OpenAPI definition (design-first approach)
+## Generate a Ballerina service 
+
+Generating a Ballerina service from an OpenAPI definition is named as the `design-first approach`. To generate only the Ballerina service, you can use the following CLI command of the OpenAPI tool.
```
Comment (lnash94) [inline learn/guides/ballerina-openapi-support.md:41]: Following the  `API design-first approach`,  you can generate only the Ballerina service from an OpenAPI definition, using the below CLI command of the OpenAPI tool.

File: learn/guides/ballerina-openapi-support.md [hunk 2]
```diff
@@ -57,26 +59,25 @@ The service generation process is complete. The following files were created.
...
 
->**Info:** For more command options, see
-[OpenAPI to Ballerina command reference](#openapi-to-ballerina-command-reference)
+>**Info:** For more command options, see [Command reference](#command-reference).
 
-## Export an OpenAPI contract from a Ballerina service (code-first approach)
+## Export an OpenAPI contract
 
-You can convert your Ballerina service APIs into human-readable or machine-readable documents such as OpenAPI documents by using the Ballerina to OpenAPI command as follows.
+Exporting an OpenAPI contract from a Ballerina service is named as the `code-first approach`. You can convert your Ballerina service APIs into human-readable or machine-readable documents such as OpenAPI documents by using the Ballerina to OpenAPI CLI Tool as follows.
```
Comment (lnash94) [inline learn/guides/ballerina-openapi-support.md:79]: Following the `API code-first approach`, you can convert your Ballerina service APIs into human-readable or machine-readable documents such as OpenAPI documents by using the Ballerina to OpenAPI CLI Tool as follows.

File: learn/guides/ballerina-openapi-support.md (hunk 1 above)
Comment (lnash94) [reply learn/guides/ballerina-openapi-support.md:41]:
```suggestion
Following the  `API design-first approach`,  you can generate only the Ballerina service from an OpenAPI definition, using the below CLI command of the OpenAPI tool.
```

File: learn/guides/ballerina-openapi-support.md (hunk 2 above)
Comment (lnash94) [reply learn/guides/ballerina-openapi-support.md:79]:
```suggestion
Following the `API code-first approach`, you can convert your Ballerina service APIs into human-readable or machine-readable documents such as OpenAPI documents by using the Ballerina to OpenAPI CLI Tool as follows.
```

**Comment by praneesha** [comment]
The above review comments will be addressed in https://github.com/ballerina-platform/ballerina-dev-website/pull/4162.

**Pull request merged**